app/decorators/cd_api/hearing_decorator.rb

```diff
@@ -6,5 +6,41 @@ def cracked_ineffective_trial
       @cracked_ineffective_trial ||= decorate(object.hearing.cracked_ineffective_trial,
                                               CdApi::CrackedIneffectiveTrialDecorator)
     end
+
+    def defence_counsels_list
+      # TODO: Add logic of attendance days for each defence
```

Comment:
Can you confirm we have a ticket associated with this TODO please

Reply:
Yep, it's the ticket [AAC-828](https://dsdmoj.atlassian.net/browse/AAC-828?atlOrigin=eyJpIjoiZTE4MGNiZmE2NjZiNDVhNmE1MGZlMjRjMGI2MWYzNWYiLCJwIjoiaiJ9)